bundles/org.dataflowanalysis.analysis.converter/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,22 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: org.dataflowanalysis.analysis.converter
+Bundle-SymbolicName: org.dataflowanalysis.analysis.converter
+Bundle-Version: 1.0.0.qualifier
+Export-Package: org.dataflowanalysis.analysis.converter,
+ org.dataflowanalysis.analysis.converter.microsecend,
+ org.dataflowanalysis.analysis.converter.webdfd
+Require-Bundle: org.dataflowanalysis.dfd.datadictionary;bundle-version="0.1.0",
+ org.dataflowanalysis.dfd.dataflowdiagram;bundle-version="0.1.0",
+ com.fasterxml.jackson.core.jackson-annotations;bundle-version="2.13.2",
+ com.fasterxml.jackson.core.jackson-core;bundle-version="2.13.2",
+ com.fasterxml.jackson.core.jackson-databind;bundle-version="2.13.2",
+ org.eclipse.emf.ecore.xmi;bundle-version="2.18.0",
+ junit-jupiter-api;bundle-version="5.9.2",
```

Comment:
Das Converter Plugin sollte keine Abh. auf Junit haben. Nur das Test Plugin sollte das haben. Bitte noch entfernen und schauen, dass alles baut.

Reply:
Was removed in last commit